File: filedata/entity/c#/Videostype.cs

using System;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
namespace Coldairarrow.Entity.Video
{
    /// <summary>
    /// 
    /// </summary>
    [Table("videos_type")]
    public class Videostype
    {
        
        /// <summary>
        /// 主键ID 
可关联Videos表中VideoTypeId字段
        /// </summary>
        [Column("Id")]
        public string Id { get; set; }
    
    
        /// <summary>
        /// 专属站点 不为null则为专属
        /// </summary>
        [Column("CompanyStyle")]
        public string CompanyStyle { get; set; }
    
    
        /// <summary>
        /// 视频从属类型名称
如:国产自拍,激情欧美,成人动漫 等
        /// </summary>
        [Column("TypeName")]
        public string TypeName { get; set; }
    
    
        /// <summary>
        /// 视频类型封面图 可为null
        /// </summary>
        [Column("TypeImg")]
        public string TypeImg { get; set; }
    
    
        /// <summary>
        /// 排序
        /// </summary>
        [Column("Orders")]
        public int Orders { get; set; }
    
    
        /// <summary>
        /// 状态 0=停用 1=启用
        /// </summary>
        [Column("Status")]
        public bool Status { get; set; }
    
    
        /// <summary>
        /// 视频类型详情说明
        /// </summary>
        [Column("TypeTitle")]
        public string TypeTitle { get; set; }
    
    
        /// <summary>
        /// 创建时间
        /// </summary>
        [Column("CreatTime")]
        public DateTime CreatTime { get; set; }
    
    
    }
}

File: filedata/entity/java/videosType.java

package entity;
// 
public class Videostype  extends BaseDomain{
    public Videostype() {

    }
    
    //
    private String Id;
        
    //
    private String CompanyStyle;
        
    //
    private String TypeName;
        
    //
    private String TypeImg;
        
    //
    private int(10) Orders;
        
    //
    private bit(10) Status;
        
    //
    private String TypeTitle;
        
    //
    private Date CreatTime;
        
    public String getId() {
        return Id;
    }
    public void setId(String Id) {
        this.Id = Id;
    }
            
    public String getCompanystyle() {
        return Companystyle;
    }
    public void setCompanystyle(String Companystyle) {
        this.Companystyle = Companystyle;
    }
            
    public String getTypename() {
        return Typename;
    }
    public void setTypename(String Typename) {
        this.Typename = Typename;
    }
            
    public String getTypeimg() {
        return Typeimg;
    }
    public void setTypeimg(String Typeimg) {
        this.Typeimg = Typeimg;
    }
            
    public int(10) getOrders() {
        return Orders;
    }
    public void setOrders(int(10) Orders) {
        this.Orders = Orders;
    }
            
    public bit(10) getStatus() {
        return Status;
    }
    public void setStatus(bit(10) Status) {
        this.Status = Status;
    }
            
    public String getTypetitle() {
        return Typetitle;
    }
    public void setTypetitle(String Typetitle) {
        this.Typetitle = Typetitle;
    }
            
    public Date getCreattime() {
        return Creattime;
    }
    public void setCreattime(Date Creattime) {
        this.Creattime = Creattime;
    }
             
    @Override
    public String toString() {
        return "videos_type[Id=" + Id + ",CompanyStyle=" + CompanyStyle + ",TypeName=" + TypeName + ",TypeImg=" + TypeImg + ",Orders=" + Orders + ",Status=" + Status + ",TypeTitle=" + TypeTitle + ",CreatTime=" + CreatTime + "]";
    }
        
    
}
        